draft-ietf-rats-msg-wrap.md

```diff
@@ -221,15 +221,16 @@ This always applies, even if the conceptual message format is already textual (e
 When using CBOR, the value field MUST be encoded as a CBOR byte string.
 
 `ind`:
-: An optional bitmap that indicates which conceptual message types are
+: An optional 4-byte bitmap that indicates which conceptual message types are
```

Comment:
The byte length shouldn't be given here. The CBOR integer will get typically be encoded with preferred serialization so its length will vary.

The only reason to specify a length here is so that a constrained decoder knows an upper bound. 64-bits, the max of uint, is usually not a problem for constrained environments so it might just be best to not say anything about the length in this section, leaving it up to the message types registry.

I'm also OK with 2**32 max. Can you use both .bits and .size on a uint? Also, it seems helpful to say that the max indicator is 31 in 10.5.

Reply:
@henkbirkholz provided a fix in [ba38b7d](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/217/commits/ba38b7dcc8a72da052e436844b8876e55b8bc824)